### DISCUSSION TITLE
July 20th 2023 PowerShell Community Call

### DISCUSSION THREAD

**Discussion by StevenBucher98** [PowerShell/PowerShell#19815] [Community Call]
Agenda
- PSResourceGet + Public Bug Bash @SydneyhSmith 
- Feedback providers + JSON Adapter demo @StevenBucher98 
- Docs updates @sdwheeler 
- DSC v3 Update @SteveL-MSFT 
- 7.2 7.3 Servicing release @adityapatwardhan 
- SSH Update @maertendMSFT @vthiebaut10 
- Conference recap @StevenBucher98 @SydneyhSmith 
- EditFile demo @JamesWTruher 
- Ashlee Introduction @ananze

**Comment by kilasuit** [comment]
- [ ] Update from across the Working Groups

**Comment by SteveL-MSFT** [comment]
@kilasuit can you be more specific on what you're looking for?  Each WG to give an update?

**Reply by kilasuit** [reply]
I was thinking on a instead of each WG having a single general update of how the different WG's are collaborating (frequency of when they get together & how long for) + if possible stats on number of issues/prs that each WG has been able to discuss. This would help in setting expectations to the community as I keep seeing very similar comments like `it's been xyz years since this was rasied, why isn't this fixed yet` which isn't helpful but does show some of the annoyances in the wider community. It would also allow for us to tell of anything the WG's as a collective have in the pipeline to help with WG engagement going forward. Will discuss more of this with you & others before this community call though.

**Reply by floh96** [reply]
see https://github.com/PowerShell/PowerShell/issues/19632#issuecomment-1579840921

> Maybe make their work more public? For example, use GitHub Projects so that we can see which issues are at what stage of consideration and what the results of their work are. Or they could have used more stage labels.

**Reply by SteveL-MSFT** [reply]
@floh96 I think using Projects would be a great way to show priority of how issues/PRs get reviewed by WGs.  Let me think about how to operationalize this.

**Comment by SteveL-MSFT** [comment]
Demitrius to give demo of `winget configure`

**Comment by maertendMSFT** [comment]
@StevenBucher98, @vthiebaut10 will be doing the SSH update today!

**Comment by JustinGrote** [comment]
If time I can demo some of the Pester VSCode Extension new features:
- Mac/Windows/Linux/Codespaces Support
- Continuous Run
- Refresh
- Bug Fixes

**Comment by ThomasNieto** [comment]
@StevenBucher98 when will the recording be posted?

**Reply by StevenBucher98** [reply]
We are working to get it published very soon, hopefully in the next day or so

**Reply by StevenBucher98** [reply]
Its live! https://www.youtube.com/watch?v=31WGvFd8ZZY @ThomasNieto !

**Reply by ThomasNieto** [reply]
@StevenBucher98 Thanks!
